po/meson.build

```
@@ -1,3 +1,5 @@
 i18n.gettext(gettext_name,
     args: ['--directory=' + meson.source_root(), '--from-code=UTF-8']
 )
```

While we're here we might as well update to use preset glib
```suggestion
i18n.gettext(gettext_name,
    args: '--directory=' + meson.source_root(),
    preset: 'glib'
)
```